### ISSUE TITLE
Feature-request : Favorite group in WLM

### ISSUE THREAD

**Issue by Mika56** [emesene/emesene#298]
Hi,

On the WLM network, you can add "favorite" contacts. This creates a group that is in top of the contact list.
I tried to modify the source code of emesene, but i'm not a Python coder and I couldn't do what i wanted.

In french, the group is named "Favoris", and i think it is "Favorites" in the english version.

Mika.

**Comment by c10ud** [comment]
The group is already in your address book (at least, in my account it exists and it's localized, yes).

In order to put it first, we need to find out what differentiate it between other groups apart from the localized name, and atm i don't know anything about it, it's just a group as others are

**Comment by Mika56** [comment]
Couldn't we, at least, be able to order ourselve our group list ?
I mean, be able to change manualy the group order ?

Mika.

**Comment by c10ud** [comment]
they're ordered by name iirc, so just create a group with a proper name like "0 - Friends" or something.. i can't think of any way on implementing a heavy custom sorting

**Comment by marianoguerra** [comment]
I implemented group and contact "weights" in the contact list, they don't have a frontend but you can use it to give a given group more weight than others so it stays on the top, the idea can be used with different criterias like amount of messages inetrchanged with that user and so on.

tl;dr the basic is implemented, we need a frontend for it.
